indexd/index/schema.py

```diff
@@ -54,7 +54,7 @@
                 "sha256": {"type": "string", "pattern": "^[0-9a-f]{64}$"},
                 "sha512": {"type": "string", "pattern": "^[0-9a-f]{128}$"},
                 "crc": {"type": "string", "pattern": "^[0-9a-f]{8}$"},
-                "etag": {"type": "string", "pattern": "^[0-9a-f]{32}(-\d+)?$"},
+                "etag": {"type": "string", "pattern": "^[0-9a-f]{32}(-\\d+)?$"},
```

Comment:
i don't think that's correct, we want `\d` (a digit), not `\\d` (literally the characters "/d"), why did we need this change? 🤔

Reply:
I think you're right, I changed it because of a warning about future deprecation, but now it's back to how it was before, we'll just need to change it in a future upgrade for python

https://docs.python.domainunion.de/3/library/re.html